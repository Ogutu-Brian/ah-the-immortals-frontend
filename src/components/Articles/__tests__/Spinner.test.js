import {mount} from 'enzyme';
import React from 'react';
import Spinner from '../../../views/Article/Spinner';



describe('pass spinner feed', () => {
  const wrapper = mount(
    <Spinner />
  );
  it('test post feed ', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
