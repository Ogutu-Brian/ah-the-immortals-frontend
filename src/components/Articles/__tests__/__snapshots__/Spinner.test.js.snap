// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pass spinner feed test post feed  1`] = `ReactWrapper {}`;
